packages/orchestration/README.md

```diff
@@ -276,23 +287,19 @@ try {
 }
 ```
 
+#### Error Handling
+
 Both `chatCompletion()` and `getContent()` methods can throw errors.
 
-- **axios errors**:  
+- **Axios Errors**:  
   When the chat completion request fails with a `400` status code, the caught error will be an `Axios` error.
   The property `error.response.data.message` may provide additional details about the failure's cause.
 
-- **output content filtered**:  
-  The method `getContent()` can throw an error if the output filter filters the model output.
+- **Output Content Filtered**:  
+  The `getContent()` method can throw an error if the output filter filters the model output.
```

Comment:
[q] can or does?

Reply:
It only throws an error for the case when the output gets filtered. So, I would like to leave it as can.